### PR TITLE
[IMG-175] Windows support policy

### PR DESCRIPTION
**Pull request by JalexChen**
# Description
A brief description of the changes.
Adds Windows Image Support Policy to docs

# Reasons
[IMG-175](https://circleci.atlassian.net/jira/software/c/projects/IMG/boards/683?selectedIssue=IMG-175)

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [N/A] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

[IMG-175]: https://circleci.atlassian.net/browse/IMG-175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by JalexChen** [comment]
@sebastian-lerner @brentmmarks @ryanbourdais - do you mind taking a look through this?

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
```
Comment (brentmmarks) [inline jekyll/_cci2/windows-images-support-policy.adoc:5]: Same question about server?

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+We support various tags for the Windows environment, allowing you to choose a suitable image for your needs.
+
+- `default`: This image is the current stable image. Jobs will default to this if no image is specified.
+
+- `current`: This version of Windows will receive updates approximately every 3 months. If no tag is provided, `current` will be the version that jobs default to.
+
+- `edge`: This tag is reserved for previews of new releases, which will initially point to this tag. The tag may include incremental updates relative to the current quarterly image release, which may change without notice, and is not recommended for production CI workloads. The `current` tag is promoted from `edge`
+
+- `previous`: Once an `edge` image is promoted to `default` and `current`, the previous `default` and `current` image is moved to the `previous` tag.
```
Comment (brentmmarks) [inline jekyll/_cci2/windows-images-support-policy.adoc:38]: previous version of Windows Server will receive the previous version of current.

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+[#tagging]
+== Tagging
+
+We support various tags for the Windows environment, allowing you to choose a suitable image for your needs.
+
+- `default`: This image is the current stable image. Jobs will default to this if no image is specified.
+
+- `current`: This version of Windows will receive updates approximately every 3 months. If no tag is provided, `current` will be the version that jobs default to.
+
+- `edge`: This tag is reserved for previews of new releases, which will initially point to this tag. The tag may include incremental updates relative to the current quarterly image release, which may change without notice, and is not recommended for production CI workloads. The `current` tag is promoted from `edge`
```
Comment (brentmmarks) [inline jekyll/_cci2/windows-images-support-policy.adoc:36]: tags are reserved for previews of new releases, which will initially point to this tag. The edge tags may include incremental updates to the current quarterly image release, which may change without notice, and is not recommended to be used for production CI workloads. currentwill be updated with these changes after period of stability.

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+The Windows images are based on Windows Server. We aim to maintain and release two major versions of Windows Server at all times. Currently we support `Windows server 2019` and `Windows server 2022`
+
+Tags will be redirected to the updated images automatically as described in the xref:#tagging[tagging] section of this document. We will announce these releases on our link:https://discuss.circleci.com/[Discuss Forum].
+
+[#tagging]
+== Tagging
+
+We support various tags for the Windows environment, allowing you to choose a suitable image for your needs.
+
+- `default`: This image is the current stable image. Jobs will default to this if no image is specified.
```
Comment (brentmmarks) [inline jekyll/_cci2/windows-images-support-policy.adoc:32]: version is what you will receive if do not specify an image.

File: jekyll/_cci2/windows-images-support-policy.adoc (hunk 2 above)
Comment (JalexChen) [reply jekyll/_cci2/windows-images-support-policy.adoc:38]: will do

File: jekyll/_cci2/windows-images-support-policy.adoc (hunk 3 above)
Comment (JalexChen) [reply jekyll/_cci2/windows-images-support-policy.adoc:36]: will do

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+- `current`: This version of Windows will receive updates approximately every 3 months. If no tag is provided, `current` will be the version that jobs default to.
```
[the remark this answers is not in the document]
Comment (JalexChen) [reply jekyll/_cci2/windows-images-support-policy.adoc:34]: will do

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+[cols=2*, options="header", stripes=even]
+|===
+| Version
+| Support
+
+| Windows Server 2019
+| Build quarterly images and retain Q4 (October) releases from previous two years
+
+| Windows Server 2022
+| Build quarterly images and Q4 (October) releases retained
```
Comment (brentmmarks) [inline jekyll/_cci2/windows-images-support-policy.adoc:77]: We wanted specific wording for this to show we will only keep Q4 image each year: `Quarterly images built and only Q4 image retained each year`

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+| Support
+
+| Windows Server 2019
+| Deprecated and removed entirely
+
+| Windows Server 2022
+| Retain only current and past two Q4 (October) releases
+
+| Windows Server 2025
+| Quarterly images built and only Q4 (October) image retained
```
Comment (brentmmarks) [inline jekyll/_cci2/windows-images-support-policy.adoc:95]: `Quarterly images built and only October (Q4) image retained each year`

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+[cols=2*, options="header", stripes=even]
+|===
+| Version
+| Support
+
+| Windows Server 2019
+| Build quarterly images and retain Q4 (October) releases from previous two years
+
+| Windows Server 2022
+| Build quarterly images and only Q4 (October) releases retained each year
```
Comment (brentmmarks) [inline jekyll/_cci2/windows-images-support-policy.adoc:77]: Please remove build

Comment (JalexChen) [reply jekyll/_cci2/windows-images-support-policy.adoc:77]: This is what is on the policy doc

Comment (brentmmarks) [reply jekyll/_cci2/windows-images-support-policy.adoc:77]: Windows Server 2019 | Retain only Q4 (October) releases (total of 2 images: one image each from 2021, 2022)Quarterly images built and only Q4 image retained each year
-- | --

Comment (brentmmarks) [reply jekyll/_cci2/windows-images-support-policy.adoc:77]: the general wording has changed. Am ok with this for this one, but moving forward we need to keep something like this where we have approval to just copy paste

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+:page-layout: classic-docs
+:page-liquid:
+:page-description: CircleCI Windows images release, update, and deprecation policy
+:icons: font
+:experimental:
+
+[#overview]
+== Overview
+
+This document outlines the CircleCI Windows image release, update, and deprecation policy. This policy applies to all Windows images built for the Windows Executor.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:16]:
```suggestion
This document outlines the CircleCI Windows image release, update, and deprecation policy. This policy applies to all Windows images built for the Windows executor.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+
+[#release-policy]
+== Release policy
+
+The Windows images are based on Windows Server. We aim to maintain and release two major versions of Windows Server at all times. Currently we support `Windows server 2019` and `Windows server 2022`
+
+Tags will be redirected to the updated images automatically as described in the xref:#tagging[tagging] section of this document. We will announce these releases on our link:https://discuss.circleci.com/[Discuss Forum].
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:23]:
```suggestion
Tags will be redirected to the updated images automatically as described in the <<tagging>> section of this document. We will announce these releases on our link:https://discuss.circleci.com/[Discuss Forum].
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+
+[#tagging]
+== Tagging
+
+We support various tags for the Windows environment, allowing you to choose a suitable image for your needs.
+
+- `default`: version is what you will receive if do not specify an image.
+
+- `current`: version of Windows Server will receive updates approximately every 3 months. This tag will be the same version as what is provided if no tag is provided.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:32]:
```suggestion
- `current`: version of Windows Server will receive updates approximately every three months. This tag will be the same version as what is provided if no tag is specified.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+
+- `edge`: tags are reserved for previews of new releases, which will initially point to this tag. The `edge` tags may include incremental updates to the current quarterly image release, which may change without notice, and is not recommended to be used for production CI workloads. `current` will be updated with these changes after period of stability.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:34]:
```suggestion
- `edge`: tags are reserved for previews of new releases, which will initially point to this tag. The `edge` tags may include incremental updates to the current quarterly image release, which may change without notice, and is not recommended to be used for production CI workloads. `current` will be updated with these changes after a period of stability.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+
+- `previous`: version of Windows Server will receive the previous version of `current`.
+
+We generally install the latest Major version of Visual Studio Community Edition that coincides with the Windows Server version in the image when it is built.
+
+We always aim to release a version of the supported windows images (e.g. currently this is `Windows server 2019` and `Windows server 2022`) every 3 months. We recommend using the `default` version and not pinning to a date version.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:40]:
```suggestion
We always aim to release a version of the supported windows images (for example, currently this is `Windows server 2019` and `Windows server 2022`) every 3 months. We recommend using the `default` version and not pinning to a date version.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+
+== Tagging with the Windows Orb:
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:42]:
```suggestion
[#tagging-with-the-windows-orb]
== Tagging with the Windows orb
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+
+The variant of Windows Server can be specified by choosing the name parameter in the Windows Orb. The most recent version of Windows Server will always be accessible via the `win/default` name, but previous versions require explicit definition, such as `win/server-2019`.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:44]:
```suggestion
The variant of Windows Server can be specified by choosing the name parameter in the Windows orb. The most recent version of Windows Server will always be accessible via the `win/default` name, but previous versions require explicit definition, such as `win/server-2019`.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+
+Each quarterly (published every 3 months) image for the current tag Windows Server version is published to a date-based tag, such as 2023.10.1, which can be explicitly defined in the version parameter of the Orb.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:46]:
```suggestion
Each quarterly (published every 3 months) image for the `current` tag Windows Server version is published to a date-based tag, such as `2023.10.1`, which can be explicitly defined in the version parameter of the orb.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 17]
```diff
@@ -0,0 +1,105 @@
...
+
+[#critical-cve-patches]
+== Critical CVE patches
+
+When critical CVEs are disclosed that affect the versions of the operating system or software stack in our Windows Images, we will investigate the impact that this has on our images being used within the CircleCI execution environment. If customers are impacted by these CVEs we will push a patch fix to the released image(s), this image will supersede the original image.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:51]:
```suggestion
When critical CVEs are disclosed that affect the versions of the operating system or software stack in our Windows images, we will investigate the impact that this has on our images being used within the CircleCI execution environment. If customers are impacted by these CVEs we will push a patch fix to the released image(s). This image will supersede the original image.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
+
+[#bug-reports-issues-and-prs]
+== Bug reports, issues, and PRs
+
+You can file a link:https://support.circleci.com/hc/en-us/requests/new[support ticket] with CircleCI Support for any issues or bugs found with the Windows images. Our support team will be able to escalate issues internally to the correct engineering team and provide updates on the issue.
+
+[#image-lifespan-eol]
+== Image lifespan / EOL
+
+Once the next major version of Windows Server is released by Microsoft and generally available to customers, we will deprecate the oldest version of Windows Server that we support and schedule it to be removed.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:61]:
```suggestion
Once the next major version of Windows Server is released by Microsoft, and generally available to customers, we will deprecate the oldest version of Windows Server that we support and schedule it to be removed.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
+
+After the new version of Windows Server has been released, we will then move the current Windows Server version into an extended support phase and move to the tagging system described in the Tagging section of this document.
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:63]:
```suggestion
After the new version of Windows Server has been released, we will then move the current Windows Server version into an extended support phase and move to the tagging system described in the <<tagging>> section of this document.
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 20]
```diff
@@ -0,0 +1,105 @@
...
+
+Current Deprecation (Jan 2024):
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:65]:
```suggestion
Current deprecation (Jan 2024):
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 21]
```diff
@@ -0,0 +1,105 @@
...
+| Support
+
+| Windows Server 2019
+| Build quarterly images and retain Q4 (October) releases from previous two years
+
+| Windows Server 2022
+| Build quarterly images and only Q4 (October) releases retained each year
+|===
+
+Example: When Windows Server 2025 is Released
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:80]:
```suggestion
Example: When Windows Server 2025 is released
```

File: jekyll/_cci2/windows-images-support-policy.adoc [hunk 22]
```diff
@@ -0,0 +1,105 @@
...
+| Windows Server 2022
+| Retain only current and past two Q4 (October) releases
+
+| Windows Server 2025
+| Quarterly images built and only Q4 (October) image retained each year
+|===
+
+When an image is selected for deprecation and removal, we will create an announcement on our Discuss forum, along with reaching out via email to developers who have requested one of the deprecated images in their recent jobs.
+
+We will also plan brownouts to ensure users are aware of the approaching removal of deprecated images. Generally, we will aim to start an EOL process within 3 months of a new version release
```
Comment (rosieyohannan) [inline jekyll/_cci2/windows-images-support-policy.adoc:100]:
```suggestion
We will also plan brownouts to ensure users are aware of the approaching removal of deprecated images. Generally, we will aim to start an EOL process within 3 months of a new version release.
```

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**